lua/nvim-treesitter/configs.lua

```diff
@@ -280,13 +287,6 @@ function M.setup(user_data)
   end
 
   config.modules.ensure_installed = nil
-
-  recurse_modules(function(_, _, new_path)
-    local data = utils.get_at_path(config.modules, new_path)
-    if data.enable then
-      enable_all(new_path)
-    end
-  end, config.modules)
```

Comment:
If this code would be important it would be actually executed. Might be in some configs. But each module defines an autocmd at init to actually enable it.